src/scripts/main.js

```diff
@@ -1,3 +1,86 @@
 'use strict';
 
-// write code here
+const grid = document.querySelector('.field');
+const gridRows = grid.rows;
+const gridBody = document.querySelector('tr').parentNode;
+
+const addCellButton = document.querySelector('.append-column');
+const removeCellButton = document.querySelector('.remove-column');
+const addRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+
+addCellButton.addEventListener('click', () => {
+  [...gridRows].forEach(row => {
+    const newCell = document.createElement('td');
+
+    row.append(newCell);
+  });
+
+  if (gridBody.rows[0].cells.length > 9) {
+    addCellButton.setAttribute('disabled', 'true');
+  }
+
+  if (
+    gridBody.rows[0].cells.length > 2
+    && removeCellButton.hasAttribute('disabled')
+  ) {
+    removeCellButton.removeAttribute('disabled');
+  }
+});
+
+removeCellButton.addEventListener('click', () => {
+  const currentCells = allCells('td');
+  const step = gridBody.rows[0].cells.length;
+
+  for (let i = 0; i < currentCells.length; i += step) {
+    currentCells[i].remove();
+  }
+
+  if (gridBody.rows[0].cells.length < 3) {
+    removeCellButton.setAttribute('disabled', 'true');
+  }
+
+  if (
+    gridBody.rows[0].cells.length < 10
+    && addCellButton.hasAttribute('disabled')
```

Comment:
It would be more readable
```suggestion
  if (isMaxColumn && isCellButtonDisabled)
```